play.py

```diff
@@ -45,16 +67,17 @@ def do_GET(self):
         output = [
             int(joystick[0] * 80),
             int(joystick[1] * 80),
-            int(round(joystick[2])),
-            int(round(joystick[3])),
-            int(round(joystick[4])),
+            int(joystick[2]),
+            int(joystick[3]),
+            int(joystick[4]),
+            #int(round(joystick[5])),
         ]
 
         ### print to console
         if (manual_override):
             cprint("Manual: " + str(output), 'yellow')
         else:
-            cprint("AI: " + str(output), 'green')
+            cprint("AI: " + str(output) + str(p2), 'green')
```

Comment:
why are you printing this value here? Isn't it already captured in the output variable?